Documentation/Contributing.md

```diff
@@ -0,0 +1,283 @@
+# Contributing to Swift Package Manager
+There are several types of contributions one can make. Bug fixes, documentation and enhancements that do not materially change the user facing semantics of Swift Package Manager should be submitted directly as PR.
+
+Larger changes that do materially change the semantics of Swift Package Manager (e.g. changes to the manifest format or behavior) are required to go through [Swift Evolution Process](https://github.com/apple/swift-evolution/blob/master/process.md).
+
+To see how previous evolution decisions for SwiftPM have been made and have some direction for the development of future features please check out the [Community Proposals](https://forums.swift.org/tag/packagemanager).
+
+For more information about making contributions to the Swift project in general see [Swift Contribution Guide](https://swift.org/contributing/).
+
+## Reporting issues
+* [SwiftPM JIRA Bug Tracker](https://bugs.swift.org/browse/SR-13640?jql=component%20%3D%20%22Package%20Manager%22)
+* [Guide for filing quality bug reports](https://github.com/apple/swift-package-manager/blob/main/Documentation/Resources.md#reporting-a-good-swiftpm-bug)
+
+Report a bug guide: https://github.com/apple/swift-package-manager/blob/main/Documentation/Resources.md#reporting-a-good-swiftpm-bug.  
+JIRA Bug Tracker (a place where you can open bugs, enhancements to start to contribute): [https://bugs.swift.org/browse/SR-13640?jql=component%20%3D%20%22Package%20Manager%22](https://bugs.swift.org/browse/SR-13640?jql=component%20%3D%20%22Package%20Manager%22).
+
+## Development environment
+
+SwiftPM is typically built with a pre-existing version of SwiftPM present on the system, but there are have multiple ways to setup your development environment:
+
+### Using Xcode (Easiest)
+
+1. Install Xcode from [https://developer.apple.com/xcode](https://developer.apple.com/xcode) (including betas!).
+2. Open SwiftPM's `Package.swift manifest with Xcode.
```

Comment:
remove this line, we are just talking about setting up the environment here. we could replace it with an explanation on how to verify the right Xcode is installed